packages/aws-fargate/images/instana-aws-fargate/build-and-push.sh

```diff
@@ -39,24 +55,17 @@ npm_tag=$2
 
 ./build.sh $build_mode $npm_tag
 
-setImageTag $image_tag_prefix $build_mode
+setImageTag $image_tag_prefix $build_mode $npm_tag
```

Comment:
I refactored our most recent modifications a bit so that $image_tag now also contains the Docker tag (`:latest` or `:next`). This made it possible to remove most of the if-else snippets we added earlier. I think it is a bit more maintainable/readable now. (Same in build.sh)